scripts/install-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+go get sigs.k8s.io/kind@v0.3.0
+# TODO(mengqiy): use a released version after v2.1.0 is out.
+go get sigs.k8s.io/kustomize@f9c631e9eec7a2d6e46eb9e1bf5122f68b97d12d
+
+# You can use --image flag to specify the cluster version you want, e.g --image=kindest/node:v1.13.6, the supported version are listed at https://hub.docker.com/r/kindest/node/tags
+kind create cluster --config test/kind-config.yaml --image=kindest/node:v1.14.1
```

Comment:
you should name this script something else if it's creating a cluster -- `install-dependencies` implies it's just installing things.